src/Client/Web/Bit.Client.Web.BlazorUI.Tests/SearchBoxes/BitSearchBoxTests.cs

```diff
@@ -37,7 +37,7 @@ public void SearchBox_NoAnimation_ShouldHaveClassName(bool disableAnimation)
         {
             var component = RenderComponent<BitSearchBoxTest>(parameter =>
             parameter.Add(p => p.DisableAnimation, disableAnimation));
-            var searchBox = component.Find(".bit-search-box-container");
+            var searchBox = component.Find(".bit-srch-box-container");
```

Comment:
"sch" or any other 3-letters abbreviation

Reply:
bit-search-box-container changed to bit-sch-box-container